openedx/features/course_experience/__init__.py

```diff
@@ -25,6 +25,9 @@
 # Waffle flag to enable a review page link from the unified home page
 SHOW_REVIEWS_TOOL_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'show_reviews_tool')
 
+# Waffle flag to switch between the 'welcome message' and 'latest update' on the course home page.
+LATEST_UPDATE_FLAG = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'latest_update')
```

Comment:
@edx/learner-mercury: We should think through if this is the best way to achieve this type of flag.  Please read the comment below for details.

@dianakhuang: Please add the following comment:
```
Important Admin Note: This is meant to be configured using waffle_utils course
  override only.  Either do not create the actual waffle flag, or be sure to unset the
  flag even for Superusers.
```